### DISCUSSION TITLE
Company icons

### DISCUSSION THREAD

**Discussion by tembbo** [lucide-icons/lucide#2696] [Ideas]
Do you guys think it would be nice to have a own library (similar to `lucide-lab`) for company icons. It would take in the removed and deprecated icons in to its own package. Then the website could have a checkbox to include the logos from the separate package.

**Comment by karsa-mistmere** [comment]
> Do you guys think it would be nice to have a own library (similar to `lucide-lab`) for company icons. It would take in the removed and deprecated icons in to its own package. Then the website could have a checkbox to include the logos from the separate package.

Icons we deprecate are going to be removed for a reason; we will not include any libraries that feature unlicensed copyrighted content or are in any other sort of legal violation, since then by extension we would also be in violation of these copyrights.

**Reply by tembbo** [reply]
Thanks, one question though, how does libs like simple-icons work then

**Reply by karsa-mistmere** [reply]
> Thanks, one question though, how does libs like simple-icons work then

How? They sometimes just don't: https://github.com/simple-icons/simple-icons/issues/11236

Pretty much every single icon featured in Simple Icons could be forcibly removed at any moment, it's entirely dependent on the goodwill of these companies that they aren't.
